modules/ding_event/ding_event.module

```diff
@@ -382,6 +413,48 @@ function ding_event_preprocess_views_exposed_form(&$variables) {
         $variables['widgets'][$widget]->widget = $render;
       }
     }
+
+    if ($form['#info']['filter-combine']) {
+      $widget = $form['#info']['filter-combine']['value'];
+      $item = $form[$widget];
+      $render = theme('views_exposed_form_widgets_title', [
+          'element' => [
+            'label' => $variables['widgets']['filter-combine']->label,
+            'field' => $item,
+          ],
+        ]
+      );
+      unset($variables['widgets']['filter-combine']->label);
+      $variables['widgets']['filter-combine']->widget = $render;
+    }
```

Comment:
This one renders a search field to search for events, however it's hidden via css.
![Uploading image.png…]()

Reply:
Not sure I understand. This field is present and shown in main distro (however not with autocomplete).

Reply:
Okay - so it re-renders this field without a label.

Reply:
Then I don't think this should be merged unless it has some positive side effect?

Reply:
Same as above - virtually any preprocess and/or form_alter hook can be implemented in the theme to achieve same result.

Reply:
Okay, let's not port this then.